### DISCUSSION TITLE
Didn't get the student benifits

### DISCUSSION THREAD

**Discussion by lowkey999-netizen** [community/community#135735] [GitHub Education]
### Hi everyone,

This is my first time writing on this community, so please excuse me for any mistakes in this message.
I submitted my student ID as proof for the GitHub copilot on Aug 6th, but I still have not received the access. please resolve this issue.

**Comment by davevad93** [comment]
Hi @Yuanlong999 ,

What's the state of your application? if it says "`Congrats! Your request for @Yuanlong999 was approved and your benefits will be available soon.`" means you still have to wait, otherwise if it says "`Congrats! Your request for @Yuanlong999 was approved and your benefits are now available.`" means that you should have access to your benefits, in which case try the following:

Go to your [education page](https://education.github.com/learner/learn), scroll down a bit to the `Student Developer Pack offers` section, click `+` to browse all the offers and select the ones you are interested in, if you can access them correctly it means you have full access to your student benefits.

![000000](https://github.com/user-attachments/assets/33aca3c6-4ade-4b23-b2e7-93b311324db5)

If you're unable to access any of the offers, you should contact Education support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your issue.

EDIT: The pack offers page is returning a 404 error, the education platform may be experiencing issues today and that may be why you haven't been able to access your benefits yet, try waiting a while until it works again.

**Reply by davevad93** [reply]
However, the link for the `Student Developer Pack` offers is working, check it out [here](https://education.github.com/pack), you can explore the available offers and access your benefits.

**Comment by prabhjotschugh** [comment]
Hi
After approval, it can take about 5-7 days to receive the benefits. You will get an email regarding the same.
Hope it helps!
